client/events.lua

```diff
@@ -1,10 +1,18 @@
 -- Player load and unload handling
-RegisterNetEvent('QBCore:Client:OnPlayerLoaded', function()
+RegisterNetEvent('qbx_core:server:playerLoaded', function(val)
+    local invokingResource = GetInvokingResource()
+    if invokingResource and invokingResource ~= GetCurrentResourceName() then return end
+
     ShutdownLoadingScreenNui()
+    QBX.PlayerData = val
     QBX.IsLoggedIn = true
     if not Config.Server.PVP then return end
     SetCanAttackFriendly(cache.ped, true, false)
     NetworkSetFriendlyFireOption(true)
+
+    --backward compatibility
+    TriggerEvent('QBCore:Client:OnPlayerLoaded')
+    TriggerServerEvent('QBCore:Server:OnPlayerLoaded')
```

Comment:
What's the benefit of naming a new event if we'll need to continue to trigger these for backwards compatibility reasons? Why not continue to use the same event name?